### DISCUSSION TITLE
Bad news! 😫

### DISCUSSION THREAD

**Discussion by siliconApricot** [jamjnsn/cartridge#4] [Announcements]
Nearing a potential usable release, I've encountered issue after issue working with my current approach. The architecture is already becoming less and less workable, and the implementation of Vite and Express with TypeScript has not worked out well due to my own inexperience with these technologies.

I'll be spending some time exploring some frameworks to make yet another approach.  For those interested in using the project ASAP, it's fairly usable as a basic library browser right now using `yarn dev`, but I will not be able to provide any troubleshooting assistance for deploying the app in this way. Once I have some momentum, I'll move the current codebase to a branch and reset the main branch.

Sorry for anybody who was hoping for a release any time soon, but I have to do this right!

**Comment by itdraak** [comment]
"A delayed game is eventually good, but a rushed game is forever bad." 
take your time, we can wait.

**Comment by gantoine** [comment]
Hey @jamjnsn, we've been building something similar for the last couple months, [RomM](https://github.com/zurdi15/romm) 🎮 

There's a lot left to do, and we can always use the extra help! Have a look at the project/codebase, and if you're interested in helping out (and you're comfortable with the tech), let me know. 🚀
